File: py/Level1/내적.py
#
# [level 1] 내적
# https://school.programmers.co.kr/learn/courses/30/lessons/70128

# 결과
# 정확성: 100.0
# 합계: 100.0 / 100.0

def solution(a, b):
    result = 0
    for i in range(len(a)):
        result += a[i] * b[i]

    return result

File: js/lv1/내적.js
//[level 1] 내적 70128
//https://school.programmers.co.kr/learn/courses/30/lessons/70128

//결과 
//정확성: 100.0
//합계: 100.0 / 100.0

function solution(a, b) {
    var answer = 0
    for (let i = 0; i < a.length; i++) answer += a[i] * b[i];
    
    return answer;
}